src/Synergism.ts

```diff
@@ -875,6 +875,18 @@ export const player: Player = {
     ambrosiaQuarkLuck1: new BlueberryUpgrade(blueberryUpgradeData.ambrosiaQuarkLuck1, 'ambrosiaQuarkLuck1')
   },
 
+  blueberryLoadouts: {
```

Comment:
not a suggestion but here's where the 2 are being set onto player. This is why they *shouldn't* have to be added to checkvariables.